frontend/packages/console-shared/src/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './types';
 export * from './utils';
 export * from './hooks';
 export * from './sorts';
+export * from './co-fetch';
```

Comment:
This is causing an error too.. you deleted the file but not the reference.

Reply:
Argh! fixed